### DISCUSSION TITLE
AWS infra scanning with AWS Tags

### DISCUSSION THREAD

**Discussion by manish7992** [prowler-cloud/prowler#1613] [Q&A] [answered]
Hi team , 
How we can scan aws infra with some aws tags ?

**Comment by toniblyx** [comment, marked as answer]
Hi @manish7992, 
As commented in the **What's comming next** section [here](https://github.com/prowler-cloud/prowler/releases/tag/3.0.0,), Tags based audit (you will be able to scan only those resources with specific tags) is in our roadmap and will be available most probably Q1 2023.
